components/frontend/src/measurement/Overrun.js

```diff
@@ -65,3 +67,10 @@ export function Overrun({ metric_uuid, metric, report, measurements, dates }) {
         />
     )
 }
+Overrun.propTypes = {
+    dates: datesPropType,
+    measurements: measurementsPropType,
+    metric: metricPropType,
+    metric_uuid: string,
+    report: reportPropType,
+}
```

Comment:
```suggestion
}

```

Reply:
I've changed my VS Code settings to prevent this in the future. Should've done that before of course.